src/Symfony/Component/Mailer/Transport/Smtp/Stream/AbstractStream.php

```diff
@@ -74,7 +74,7 @@ public function readLine(): string
             return '';
         }
 
-        $line = fgets($this->out);
+        $line = @fgets($this->out);
```

Comment:
Do you really need to silence this?

Reply:
This is the line that throws the `fgets(): SSL: Connection reset by peer` error. It's not an exception - It's error catched by set_error_handler with severity of Error.

I guess that it could be catched by try catch (Error) but that would require rewrite of the current throws for timeout and EOF. I wanted to make as few changes as possible.

Should rewrite it to use try-catch instead?

Reply:
I was mistaken - The severity is not Error but just a E_WARNING. It cannot be catched by try catch (Error).
Seems like silencing is the only option then.

Reply:
Hm.. My initial thought was to keep the error/warning. Because if we silence it, the user will never see the text `SSL: Connection reset by peer`. 

> Seems like silencing is the only option then.

Yepp. Im afraid you are correct. Thank you for checking.

Reply:
Agree that the message could be useful for debugging.
We could call `error_clear_last()` before calling `@fgets()` and in case of an error obtain the message from `error_get_last()`.

Not sure what is the performance impact of that though.

Reply:
Performance is not critical here IMHO.